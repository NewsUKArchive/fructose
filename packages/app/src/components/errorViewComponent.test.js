/* globals describe it beforeEach expect */
import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import ErrorComponent from "./errorViewComponent";

configure({ adapter: new Adapter() });

describe("Error View Handler", () => {
  let wrapper;
  let error;

  beforeEach(() => {
    error = new Error("I am ERRRRRR");
    wrapper = shallow(<ErrorComponent error={error} />);
  });

  it("prints out error", () => expect(wrapper).toMatchSnapshot());
});
