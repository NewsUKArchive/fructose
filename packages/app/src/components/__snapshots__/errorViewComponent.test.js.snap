// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error View Handler prints out error 1`] = `undefined`;
